libs/designsystem/button/src/button.component.ts

```diff
@@ -43,10 +40,6 @@ export class ButtonComponent implements AfterContentInit {
 
   @Input() set attentionLevel(level: AttentionLevel | undefined) {
```

Comment:
Consider reducing this further to a simple Input decorator (no setter, getter or `_attentionLevel` needed)
```suggestion
  @Input() attentionLevel: AttentionLevel;
```

and then use `this.attentionLevel ` in `getAttentionLevelCssClass`